src/main/java/org/opentripplanner/model/plan/ScheduledTransitLeg.java

```diff
@@ -13,6 +13,7 @@
 import javax.annotation.Nullable;
 import org.locationtech.jts.geom.Coordinate;
 import org.locationtech.jts.geom.LineString;
+import org.opentripplanner.ext.fares.model.Distance;
```

Comment:
I am sorry, this creates a dependency from the core to the Sandbox - this should be avoided.

Reply:
Where should the Distance place be placed?

Reply:
My suggestion is `org.opentripplanner.framework.scalars`, but feel free to take a decissiosn on your own. We can always move it. See my comment on the class as well.

Reply:
Ok, did I understand correctly that you can take care of the moving of the class when you potentially need it elsewhere? If so, I think I'd prefer you to perform the move and the addition of the methods mentioned in the other comment. I can of course add them in this PR as well if you wish so

Reply:
I'm happy to do it. Just remove the `Distance` type from `ScheduledTransitLeg`.

I will do the rest.